tests/test_urls.py

```diff
@@ -302,28 +302,37 @@ def test_custom_404_rst_source(app, status, warning):
 @pytest.mark.sphinx(srcdir=rstsrcdir)
 def test_image_on_404_rst_source(app, status, warning):
     app.build()
+
+    # Check the image was added to the builder/environment images
+    assert 'test.png' in app.builder.images
+    assert 'test.png' in app.env.images
+
+    # Check the image was copied into the output dir
+    path = app.outdir / '_images' / 'test.png'
+    assert path.exists() == True
```

Comment:
You can omit the `= True` part, but if you want to keep it, you should use `is True`

Reply:
I changed them all!

That's why my editor was warning me about something there :)